tests/system/python/rpi/test_e2e_rpi_ephat.py

```diff
@@ -28,7 +28,7 @@
 SOUTH_PLUGIN = "envirophat"
 SVC_NAME = "Room-1"
 
-ASSET_PREFIX = "envirophat/"  # default for envirophat South plugin
+ASSET_PREFIX = "e_"  # default for envirophat South plugin
```

Comment:
This was only due to as we changed the default asset name value for plugin in https://github.com/foglamp/foglamp-south-envirophat/pull/32/files